problem/forms.py

```diff
@@ -134,3 +134,17 @@ class Meta:
                                         attrs={'class': 'form-control'})
         }
 
+
+class TagFilter(forms.Form):
+    tag_name = forms.CharField(max_length=20, label='Tag', required=False,
+            widget= autocomplete_light.TextWidget('TagAutocomplete',
+                                        attrs={'class': 'form-control'}))
+
+    def clean_tag_name(self):
+        tag_name = self.cleaned_data['tag_name'].strip()
+        if len(tag_name) == 0:
```

Comment:
Do you mean return an empty string?
I don't understand why return when the length is equal to 0